e2e/test/scenarios/dashboard/x-rays.cy.spec.js

```diff
@@ -97,6 +97,12 @@ describe("scenarios > x-rays", { tags: "@slow" }, () => {
 
   ["X-ray", "Compare to the rest"].forEach(action => {
     it(`"${action.toUpperCase()}" should work on a nested question made from base native question (metabase#15655)`, () => {
+      if (action === "Compare to the rest") {
+        cy.log(
+          "Skipping Compare to the rest test because it takes 8 minutes in ci",
+        );
+        cy.skipOn(action === "Compare to the rest");
+      }
```

Comment:
@iethree What is the plan for this? Do we create an issue and investigate why the test took too long?

I'm afraid we'll forget about it

Reply:
yeah, i'm gonna ping BEC about it, but I'm having trouble getting a good repro

Reply:
https://metaboat.slack.com/archives/C0641E4PB9B/p1718637304256719